renku/core/management/datasets.py

```diff
@@ -123,7 +129,7 @@ def with_dataset(self, name=None, identifier=None):
                 path.parent.mkdir(parents=True, exist_ok=False)
             except FileExistsError:
                 raise errors.DatasetExistsError(
-                    'Dataset with reference {} exists'.format(path.parent)
+                    'Dataset with UUID {} exists'.format(identifier)
```

Comment:
I think datasets imported from 3rd party and which are stamped with DOI will have identifiers equal to DOI structure and not UUID. So let's keep the wording as it was?

Reply:
This might be a bug though, which we should revert. Let's sync tomorrow.